src/pyBiodatafuse/utils.py

```diff
@@ -110,11 +110,11 @@ def combine_sources(bridgedb_df: pd.DataFrame, df_list: List[pd.DataFrame]) -> p
     :param df_list: list of dataframes to be combined.
     :returns: a single dataframe containing from a list of dataframes
     """
-    m = bridgedb_df[
-        (bridgedb_df["target.source"] == "Ensembl")
-        | (bridgedb_df["target.source"] == "PubChem-compound")
-    ]
-
+    #   m = bridgedb_df[
```

Comment:
Any specific reason to comment this out?

Reply:
I wanted to keep Uniprots. Related to the comment below about creating a column of uniprot identifiers, I think it can go for now.